assembl/static2/js/app/utils/globalFunctions.js

```diff
@@ -381,4 +381,8 @@ export const formatedTagList = (tags: Tags): Array<TagProps> =>
       }
       return result;
     }, [])
-    : []);
\ No newline at end of file
+    : []);
+
+// Function that checks wheter the browser in IE from v6 to v11
+// eslint-disable-next-line
+export const isIE = () => /*@cc_on!@*/ false || !!document.documentMode;
```

Comment:
Is this the best way? Common sense says use the `useragent`, and seems like `Trident` is the domain of IE 11 and below. https://stackoverflow.com/a/22242528

Reply:
I got the info from here: https://stackoverflow.com/questions/9847580/how-to-detect-safari-chrome-ie-firefox-and-opera-browser. It got 1463 upvotes...

Reply:
Alright, looks like I was wrong. Useragent can be spoofed. Vas y